scripts/package-distribution.ps1

```diff
@@ -19,6 +19,8 @@ $ArtifactsBinDir = Join-Path $BaseArtifactsDir "bin"
 # All direct subfolders are OS's
 $Platforms = Get-ChildItem -Path $ArtifactsBinDir
 
+$Version = "1.10.0"
```

Comment:
Is there a way we can keep this in sync with the version in the source code? Or a script to update both together?

Reply:
The update-version script was updated to update this.

Reply:
ah, I see that now.